<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(170, 255, 255);

	

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BackHome">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Back Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chart">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>图表展示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serial">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>串口通信</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Data">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>文件展示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Log">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>日志文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{

	//background-image: url(:/img/C:/Users/dell/Pictures/Uplay/新西兰的萨瑟兰瀑布和奎尔湖.jpg);

}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="first_page"/>
      <widget class="QWidget" name="chart_2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <widget class="QRadioButton" name="c1">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 1</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="c2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 2</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="c3">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 3</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="c4">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 4</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="c5">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 5</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="c6">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>chart 6</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QChartView" name="chart_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="serial_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>串口选择</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudSelect">
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="serialSelect"/>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Link">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CutLink">
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="TextReceive">
            <property name="text">
             <string>接收开关</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="TextClear">
            <property name="text">
             <string>清空接收</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>接收窗口</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>878</width>
             <height>367</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="ReceiveView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>发送窗口</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="SendView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SendText">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QListWidget" name="DataListView">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>560</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="openFile">
              <property name="text">
               <string>打开文件夹</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearFile">
              <property name="text">
               <string>清空列表</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LogWidget">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>878</width>
             <height>512</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTextEdit" name="debugEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
